src/components/file-viewer/file-viewer.e2e.ts

```diff
@@ -0,0 +1,139 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+// Since we are currently using a pre-release of Stencil to get access
+// to Puppeteer, documentation is lacking.
+//
+// Methods and properties for E2EElement (v0.13.0-9):
+// https://github.com/ionic-team/stencil/blob/a0250ffcbf5a2c657475a05052eac3a4690809d2/src/testing/puppeteer/puppeteer-declarations.ts#L78
+//
+// Matchers (expect-methods) for E2EElement (v0.13.0-9):
+// https://github.com/ionic-team/stencil/blob/a0250ffcbf5a2c657475a05052eac3a4690809d2/src/declarations/testing.ts#L5
```

Comment:
this tes file is old, 

1. I don't know how to write these tests. so someone should check it out.
2. We also need some .specx tests

Reply:
not sure if the comment above is relevant anymore

Reply:
No, this comment is way old. Please remove it 👍